tests/modules/atomicslotmigration.c

```diff
@@ -272,8 +337,27 @@ void clusterTrimEventCallback(RedisModuleCtx *ctx, RedisModuleEvent e, uint64_t
     if (e.id == REDISMODULE_EVENT_CLUSTER_SLOT_MIGRATION_TRIM) {
         /* Log the event. */
         if (numClusterTrimEvents >= MAX_EVENTS) return;
+        if (numClusterAllAsmEvents >= MAX_EVENTS) return;
         RedisModuleClusterSlotMigrationTrimInfo *info = data;
         clusterTrimEventLog[numClusterTrimEvents++] = clusterTrimInfoToString(info, sub);
+        clusterAllAsmEventLog[numClusterAllAsmEvents++] = clusterTrimInfoToString(info, sub);
+    }
+}
+
+void clusterUnownedKeysEventCallback(RedisModuleCtx *ctx, RedisModuleEvent e, uint64_t sub, void *data) {
+    REDISMODULE_NOT_USED(ctx);
+    REDISMODULE_NOT_USED(data);
+
+    RedisModule_Assert(RedisModule_IsSubEventSupported(e, sub));
+
+    if (numClusterAllAsmEvents >= MAX_EVENTS) return;
+
+    if (sub == REDISMODULE_SUBEVENT_CLUSTER_UNOWNEDKEYS_DETECTED) {
```

Comment:
Here too a `switch` statement would be cleaner.

Reply:
^^

Reply:
it is just a style :), let's leave as it is